samples/monitor/create_diagnostic_setting.py

```diff
@@ -0,0 +1,230 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import os
+
+from azure.common.credentials import ServicePrincipalCredentials
+from azure.identity import EnvironmentCredential
+from azure.mgmt.eventhub import EventHubManagementClient
+from azure.mgmt.loganalytics import LogAnalyticsManagementClient
+from azure.mgmt.monitor import MonitorClient
+from azure.mgmt.logic import LogicManagementClient
+from azure.mgmt.resource import ResourceManagementClient
+from azure.mgmt.storage import StorageManagementClient
+
+
+# Cerdential for track1 sdk
+credential4track1 = ServicePrincipalCredentials(
+    client_id=os.environ.get("AZURE_CLIENT_ID"),
+    secret=os.environ.get("AZURE_CLIENT_SECRET"),
+    tenant=os.environ.get("AZURE_TENANT_ID")
+)
+
+def main():
+
+    SUBSCRIPTION_ID = os.environ.get("SUBSCRIPTION_ID", None)
+    GROUP_NAME = "testgroupx"
+    STORAGE_ACCOUNT_NAME = "storageaccountxyzxyzxx"
+    NAMESPACE = "namespacex"
+    EVENTHUB = "eventhub"
+    AUTHORIZATION_RULE = "authorizationx"
+    INSIGHT = "insightx"
+    WORKSPACE_NAME = "workspacex"
+    WORKFLOW_NAME = "workflow"
+
+    # Create client
+    resource_client = ResourceManagementClient(
+        credential=EnvironmentCredential(),
+        subscription_id=SUBSCRIPTION_ID
+    )
+    storage_client = StorageManagementClient(
+        credential=EnvironmentCredential(),
+        subscription_id=SUBSCRIPTION_ID
+    )
+    eventhub_client = EventHubManagementClient(
+        credential=EnvironmentCredential(),
+        subscription_id=SUBSCRIPTION_ID
+    )
+    monitor_client = MonitorClient(
+        credential=EnvironmentCredential(),
+        subscription_id=SUBSCRIPTION_ID
+    )
+    # use track1 sdk
+    loganalytics_client = LogAnalyticsManagementClient(
```

Comment:
we should avoid mixture usage of track 1 and track 2 together, 
for those code samples let's just show things that can only be done with track 2,

Reply:
yeah, ok. I will disable these samples.